Modules/Files/src/Files.Infrastructure/Persistence/Database/Repository/FilesRepository.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.BlobStorage;
+using Microsoft.Extensions.Options;
+using Backbone.Modules.Files.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Files.Application.Infrastructure.Persistence;
+using Backbone.Modules.Files.Domain.Entities;
+using Backbone.Modules.Files.Application.Extensions;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.Database;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Enmeshed.BuildingBlocks.Application.Extensions;
+using Enmeshed.BuildingBlocks.Application.Pagination;
+
+namespace Backbone.Modules.Files.Infrastructure.Persistence.Database.Repository;
+public class FilesRepository : IFilesRepository
+{
+    private readonly DbSet<FileMetadata> _files;
+    private readonly IQueryable<FileMetadata> _readOnlyFiles;
+    private readonly FilesDbContext _dbContext;
+    private readonly IBlobStorage _blobStorage;
+    private readonly BlobOptions _blobOptions;
+
+    public FilesRepository(FilesDbContext dbContext, IBlobStorage blobStorage, IOptions<BlobOptions> blobOptions)
+    {
+        _files = dbContext.FileMetadata;
+        _readOnlyFiles = dbContext.FileMetadata.AsNoTracking();
+        _dbContext = dbContext;
+        _blobStorage = blobStorage;
+        _blobOptions = blobOptions.Value;
+    }
+
+    public async Task Add(FileMetadata fileMetadata, byte[] body, CancellationToken cancellationToken)
```

Comment:
Can you search for occurrences of `body` in the Files module and replace them with `content` please?